counter-app/src/test/java/camp/nextstep/edu/counter/CounterViewModelTest.kt

```diff
@@ -0,0 +1,62 @@
+package camp.nextstep.edu.counter
+
+import androidx.arch.core.executor.testing.InstantTaskExecutorRule
+import com.google.common.truth.Truth.assertThat
+import org.junit.Rule
+import org.junit.Test
+
+class CounterViewModelTest {
+    @get:Rule
+    val instantExecutorRule = InstantTaskExecutorRule()
+
+    @Test
+    fun `ViewModle생성`() {
+        // given :
+
+        // when : viewModel을 선언한다.
+        val viewModel = CounterViewModel()
+        // then :
+        assertThat(viewModel).isNotNull()
+    }
```

Comment:
ViewModel을 생성하는 로직은 따로 테스트로 만들기보다, 다른 테스트를 실행시키기 위한 최소 전제 조건으로 보는게 적합해보입니다. 다음과 같이 val 프로퍼티로 선언하거나
```suggestion
class CounterViewModelTest {
    @get:Rule
    val instantExecutorRule = InstantTaskExecutorRule()
    
    val viewModel = CounterViewModel()
```

또는 Before 어노테이션을 활용하여 테스트 실행 전 초기화 단계를 명시할 수 있습니다 🙂 

```kotlin
    lateinit var viewModel: CounterViewModel

    @Before
    fun setup() {
        viewModel = CounterViewModel()
    }
```